### ISSUE TITLE
Preview disabled during adetailer pass.

### ISSUE THREAD

**Issue by magpie514** [Bing-su/adetailer#27]
I have previews enabled, showing progress for every step (0).
But during the adetailer pass at the end of regular generation, previews are completely frozen until the very last step.

I'm using the current commit (d95d82c) with the current (5ab7f213) version of sd-webui (torch2, xformers 0.0.17, video card is a 1070 and running ubuntu 22.10). 

There are no errors in stdout and everything else functions properly.

**Comment by fanyangto** [comment]
Same here. It would be ideal not only showing the preview, but also actually show the masked area and the intermediary steps, as done by the extension "face editor". The display of intermediary steps is important to show if proper sizing/parameter zooms in the face accurately.

**Comment by Bing-su** [comment]
I can see the preview, why don't you???

**Comment by magpie514** [comment]
I'd like to know that, Bing-su. It's not a deal-breaker but it'd be nice to have them.
The issue persists even with the latest updates.

Note that _normal_ previews work fine, it's just the adetailer step what doesn't show them.

**Comment by magpie514** [comment]
Even though the issue has been nulled, I managed to figure out what the problem was.

It's an issue with _timing_ apparently. I decided to increase the amount of ADetailer steps by a rather large amount and previews started to appear after 5-6 seconds, at a time where they'd be normally finished.
If I had to wager, some conversion or uploading task is holding the preview for some seconds, and most people aren't noticing it because that step is faster in their machines (my video card is, while comfortable in speed, kind of bottom line for 8GB models).

With that said, ADetailer is not at fault here, so this issue should remain as it is, but I decided to comment in case someone else has this issue. The quick answer is, alas, "it's just our hardware being a bit slow, nothing to worry about".
